java/test/buildtests.sh

```diff
@@ -0,0 +1,7 @@
+
+for I in $( find . -name "Ipocs*Test.java"); do
+  echo $I
+  TestName=$(basename $I)
+
+  echo $(  basename $I ) 
+done
```

Comment:
What does this script do and what is it used for?
We have the scripts in the folder "scripts", and this script is in the folder java/test.

I apologize for not have noticed it until now, but I haven't had time to go thru all the files in this PR until now.

Reply:
Mistake commit - should not have been there. Start of an early version of the PS script.